include/core/parameters.h

```diff
@@ -1073,6 +1073,7 @@ namespace Parameters
     int                          contact_search_frequency;
     bool                         load_particles_from_file;
     std::string                  particles_file;
+    bool                         enable_extra_sharp_interface_vtu_output_field;
```

Comment:
It would be a good idea to start grouping the parameters within this parameter block into more logical entitties. Now you have mechanical properties, numerical parameters, mesh refinement parameters, and post-processing all in a single block. It's a bit all over the place...